px-data-grid.html

```diff
@@ -1294,6 +1294,10 @@
           this._vaadinGrid._pxDataGrid = this;
           this._vaadinGrid.addEventListener('px-sorter-changed', this._onPxSorterChanged);
 
+          if (this.gridHeight != 'auto') {
```

Comment:
I don't think this is the correct place to treat this issue.

There's a resize observer that's responsible for setting the height, and it has two issues:

- It does not fire when the component is attached to the DOM and
- As you've pointed out, it flat out does not work at all on Firefox

The resize observer is where the fixes should be.